tmVhdlProducer/algodist.py

```diff
@@ -402,9 +420,24 @@
     tmEventSetup.TripleJetOvRm: kTripleJetOvRm,
     tmEventSetup.QuadJetOvRm: kQuadJetOvRm,
     tmEventSetup.AnomalyDetectionTrigger: kAnomalyDetectionTrigger,
+    tmEventSetup.Axol1tlTrigger: kAxol1tlTrigger,
+    tmEventSetup.CicadaTrigger: kCicadaTrigger,
+    tmEventSetup.TopologicalTrigger: kTopologicalTrigger,
 }
 """Dictionary for condition type enumerations."""
 
+ValidAmodels: List[str] = [
+    "v1",
+    "v3",
+]
+
+ValidTmodels: List[str] = [
+    "hh_ele_v1",
+    "hh_had_v1",
+    "hh_mu_v1",
+]
+"""Valid models for axol1tl and topological."""
+
```

Comment:
Available ML models should not be required by `algodist`, better move to `main` or `vhdlproducer` and perform a more generic validation of supported models.